package.json

```diff
@@ -29,6 +29,7 @@
   "devDependencies": {
     "@types/enzyme": "^2.7.7",
     "@types/jest": "^19.2.2",
+    "@types/velocity-animate": "^1.2.30",
```

Comment:
Remove this from devDependencies, since it exists in dependencies (below) as well.